addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3242,6 +3243,16 @@ const SnippetOptionWidget = Widget.extend({
 
         const styles = window.getComputedStyle(this.$target[0]);
 
+        // Case where cssProperty is a CSS variable.
+        if (params.cssProperty.startsWith('--')) {
+            // For the padding of the grid items (columns in grid mode).
+            if (params.cssProperty.startsWith('--grid-item-padding') && parseFloat(widgetValue) < 0) {
+                widgetValue = '0px';
+            }
```

Comment:
You can define a data-min on the we-input definition I think? if not, I don't think it's worth code to ensure we don't enter something below 0 (probably the same for all inputs in our editor panel, to check). And especially not in a generic method like here :+1: